src/apis/post/apis.ts

```diff
@@ -1,8 +1,16 @@
-import type { GetCategoriesRes, GetPostsReq, GetPostsRes } from './types'
+import type {
+  GetCategoriesRes,
+  GetPostsReq,
+  GetPostsRes,
+  GetSortOptionsReq
+} from './types'
 import { http } from '@utils/http'
 
 export const getCategories = () =>
   http.get<null, GetCategoriesRes>('/categories')
 
 export const getPosts = (params: GetPostsReq) =>
   http.get<GetPostsReq, GetPostsRes>('/posts', params)
+
+export const getSortOption = (params: GetSortOptionsReq) =>
+  http.get<GetSortOptionsReq, GetPostsRes>(`/sorts?type=${params.type}`)
```

Comment:
```suggestion
export const getSortOptions = (params: GetSortOptionsReq) =>
  http.get<GetSortOptionsReq, GetSortOptionsRes>(`/sorts?type=${params.type}`)
```
타입 추론이 안되지 않나요..?? `GetSortOptionsRes`로 변경되어야할 것 같습니다!

Reply:
`sortOptions` 카톡에서 논의했을 때, api 호출하지 않고 FE, BE에 각자 정의해둔 값을 싱크 맞춰서 사용하는 것으로 논의된 것 같습니다!

`@constants`의 `SORT_OPTIONS` 확인 부탁드립니둥!

Reply:
넵 제거 하여 constans 사용하였습니다!
[링크](https://github.com/price-offer/offer-fe/pull/174/commits/a6f681de442a40533851b770a8a71280628c236f#diff-07f2fec6d53a08a3d99917e735698ec74ee5e358fe78e5bc05e0c44a988dc5a5)